lib/features/popup-menu/PopupMenu.js

```diff
@@ -195,6 +195,37 @@ PopupMenu.prototype.open = function(target, providerId, position, options) {
   this._render();
 };
 
+/**
+ * Refresh the popup menu entries without changing the target or position.
+ */
+PopupMenu.prototype.refresh = function() {
```

Comment:
I wonder if we can we consolidate existing behavior, i.e. call `refresh` [here](https://github.com/bpmn-io/diagram-js/blob/fe6ae727ce24be96e67dbcd6b165e08a6be65c56/lib/features/popup-menu/PopupMenu.js#L88)?

Looks good otherwise.

Reply:
Makes sense, I'll add that.

Reply:
Done. ✅